### PR TITLE
Fixed an issue and cleaned up some logging

### PR DESCRIPTION
**Pull request by psav**
* Caching of module scoped fixtures was causing a big problem, this is
  going to eventually lead to some other work
* Logging of uncollect was lying, terminology was bad
* Fixing versions for flash messages

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7b37d14fe7b9679a75b970ab12cd1c2b48c8be87:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 370c97e7000f0880e56f29c665203831bda36156:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/web_ui/flash.py:12:1: F401 'upstream' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d338d49280d45002389e0bb2f9d638cb8eb9c17a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 34d51f9ba930e256053a36065fd8f0eb20f8eaa4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/automate/explorer.py [hunk 1]
```diff
@@ -335,25 +335,30 @@ def make_path(cls, *names, **kwargs):
...
+                    create_on_init=create_on_init, *names), create_on_init=create_on_init)
         else:
             return None
 
-    def __init__(self, name=None, description=None, parent=None, domain=None):
+    def __init__(self, name=None, description=None, parent=None, domain=None, create_on_init=False):
         self.name = name
         self.description = description
         self.parent = parent or (domain if isinstance(domain, Domain) else Domain.default)
+        if create_on_init:
```
Comment (seandst) [inline cfme/automate/explorer.py]: I appreciate that this is optional, but even still I'm averse to doing more than initialization in `__init__`. Is this strictly necessary, or can it go?

Comment (psav) [inline cfme/automate/explorer.py]: To get make_path to work as it was intended I can't see a way around this unless we do some thing even hackier.

**Comment by dajoRH** [comment]
Lint report for commit aa302ae9b98d08baf8c802e41ed4573aaaad2cd1:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/automate/explorer.py:329:19: E702 multiple statements on one line (semicolon)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ffe032bd6991d9515e4bca933287948e34b60afc:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**